CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 6.4.0 - 2018-10-23
+
+* Improvement: Improved error messages when using cUrl < 7.21.2 [#2108](https://github.com/guzzle/guzzle/pull/2108)
```

Comment:
why using a capital U in `cUrl` ? https://curl.haxx.se/ writes it lowercase.

Reply:
Oh. I don’t know. I’ve always written it like that. 

Part of me wants to see how it was written 10 years ago :)

I’ll update

Reply:
according to https://curl.haxx.se/docs/faq.html the original spelling is `cURL`

Reply:
Thank you